### DISCUSSION TITLE
Welcome to About_moja_global Discussions!

### DISCUSSION THREAD

**Discussion by gmajan** [moja-global/About-moja-global#118] [General]
## 👋 Welcome!
I have activated Discussions as a place to connect on issues that are not directly related to the code or the repos.
My first test is to check how many of us will reply to this msg.
I am afraid it would scatter our comms between Slack, GitHub, email, etc.
But happy to try...

**Comment by aornugent** [comment]
@sabitarao - Guy has beaten us to it! I think what we need then is to make clear what kind of questions should go here rather than on Slack.

My thinking is that GitHub discussions should work a bit like StackOverflow - more of the 'how do I do x?' questions, so that the next person can find the thread and follow along with the answers. Slack works well for conversations but it's near impossible for new users to find existing content.

Maybe we can pin a comment to the Slack #general channel and edit the readme (and or docs.moja.global)?

**Reply by aornugent** [reply]
@harshcasper @waridrox - come on over. Could you please submit pull requests to the relevant READMEs? I'll pin a message on Slack.

**Reply by waridrox** [reply]
Sure sir, could you please mention the repositories that we are targeting to include the discussions feature ?

**Reply by HarshCasper** [reply]
Once we have the repositories, we can divide the work equally and after a discussion, submit the relevant PRs 😄

**Comment by mohanm18** [comment]
[https://github.com/moja-global/About_moja_global/wiki](url) Here upon clicking on
How to make a contribution ? [https://github.com/moja-global/.github/blob/master/CONTRIBUTING.md](url)
It gives - PAGE NOT FOUND
